ababot/dir_macros/src/lib.rs

```diff
@@ -68,6 +68,37 @@ impl Parse for InvocationTarget {
     }
 }
 
+#[proc_macro]
```

Comment:
We need to revise the entire system, but this is fine for now

Reply:
Ok